src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -23,7 +23,9 @@
     private String description;
     private BigDecimal price;
     private Group groupId;
+    private List<Order> orders = new ArrayList<>();
     private List<Cart> carts = new ArrayList<>();
+    private int amount;
```

Comment:
czy chodzi tutaj o ilosc sztuk? moze lepiej zastosowac quantity, bo w pierwszym momencie myslalam, ze to podsumowanie calego koszyka i chcialam zasugerowac zmiane na BigDecimal